Description: using sdl2 system lib, instead of the bundled one
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2019-11-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/client/client.pro
+++ b/client/client.pro
@@ -34,12 +34,6 @@
 #QMAKE_CXXFLAGS_RELEASE += -Zi
 #QMAKE_LFLAGS_RELEASE += /DEBUG /OPT:REF
 
-contains(QMAKE_TARGET.arch, x86_64) {
-   LIBS += -L$${PWD}/lib64
-} else {
-   LIBS += -L$${PWD}/lib
-}
-
 LIBS += -lSDL2
 
 DEPENDPATH += \
@@ -68,10 +62,6 @@
 INCLUDEPATH += src/sound
 INCLUDEPATH += ../shared
 INCLUDEPATH += ../server/src
-INCLUDEPATH += $$(QTDIR)/include/QtCore
-INCLUDEPATH += $$(QTDIR)/include/QtGui
-INCLUDEPATH += $$(QTDIR)/include/QtNetwork
-INCLUDEPATH += $$(QTDIR)/include/QtOpenGL
 
 HEADERS += \
     ../ai/astarmap.h \
--- a/client/src/framework/gldevice.cpp
+++ b/client/src/framework/gldevice.cpp
@@ -25,7 +25,7 @@
 
 
 #ifdef Q_OS_LINUX
-#include "GL/glx.h"
+#include <GL/glx.h>
 #endif
 
 #ifdef QT_VERSION
--- a/server/server.pro
+++ b/server/server.pro
@@ -14,12 +14,9 @@
 
 INCLUDEPATH += .
 INCLUDEPATH += ../shared
-INCLUDEPATH += $$(QTDIR)/include/QtCore
-INCLUDEPATH += $$(QTDIR)/include/QtNetwork
 
 contains(DEFINES, USE_GUI) {
    QT += widgets
-   INCLUDEPATH += $$(QTDIR)/include/QtWidgets
 }
 
 win32 {
